src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -389,6 +390,32 @@ package object metadata extends DebugEnhancedLogging {
     }
   }
 
+  def filesXmlFilesHaveOnlyAllowedAccessRights(t: TargetBag): Try[Unit] = {
+    trace(())
+    for {
+      xml <- t.tryFilesXml
+      files <- getFileElements(xml)
```

Comment:
no need to wrap this in `Try`!

```suggestion
      files = xml \ "file"
```

or even inline that below...

Reply:
Unit tests expect Try